auto3dseg/algorithm_templates/segresnet/configs/hyper_parameters.yaml

```diff
@@ -23,7 +23,7 @@ cache_rate: null
 determ: false
 fold: 0
 learning_rate: 2.0e-4
-num_images_per_batch: 1
+batch_size: 1
```

Comment:
If I remember correctly, the other 3 algorithms use `num_images_per_batch`.

Reply:
it is named as "batch_size" in "train.py" of segresnet. I did not change anything yet in the training script since I am not sure if naming serves any purpose in segresnet. @myron

Reply:
I just unify the naming in "algo.py".

Reply:
yes, I've checked and I'm actually using "batch_size" everywhere, so this proposed change is correct.. but ideally we want to bring this name to be the same to all algos (either one can be ok)

Reply:
On another note, maybe we should make a note about the unified names, such as `num_epochs` and `batch_size`/`num_images_per_batch` in this or the `monai.apps.auto3dseg`

Reply:
I would propose let's fix it to `num_images_per_batch` so that the tests/tutorials work as before:

https://github.com/Project-MONAI/research-contributions/blob/c812e5f3b7a56c998035514846aa3292394c4097/auto3dseg/algorithm_templates/segresnet/scripts/segmenter.py#L712